Users/Install.rst

```diff
@@ -299,7 +299,8 @@ Installation Errors
 In case you are getting
 ``ValueError:('Expected version spec in', 'appdirs ~=1.4.0', 'at',
 ' ~=1.4.0')``, then don't panic. It happens when you are using an outdated
-version of pip that doesn't support our version specifiers yet.
+version of pip that doesn't support our `version specifiers
+<https://www.python.org/dev/peps/pep-0440/#version-specifiers>`_ yet.
 
 
     Ideally, you have to create a virtual environment with a newer pip:
```

Comment:
I click on the "View" button here on https://github.com/coala/documentation/pull/440/files , which takes me to
 
https://github.com/thisch/documentation/blob/53b01c4f8786862ad25adb5595f60ed32f842a79/Users/Install.rst

And I look at it... ;-)